src/reference/asciidoc/logging.adoc

```diff
@@ -0,0 +1,183 @@
+[[logging]]
+=== Logging Subsystem AMQP Appenders
+
+The framework provides logging appenders for several popular logging subsystems:
+
+- log4j (since Spring AMQP _version 1.1_)
+- logback (since Spring AMQP _version 1.4_)
+- log4j2 (since Spring AMQP _version 1.6_)
+
+The appenders are configured using the normal mechanisms for the logging subsystem, available properties are specified
+in the following sections.
+
+==== Common properties
+
+The following properties are available with all appenders:
+
+.Common Appender Properties
+[cols="2l,2l,4", options="header"]
+|===
+| Property
+| Default
+| Description
+
+| exchangeName
+| logs
+| Name of the exchange to publish log events to.
+
+| exchangeType
+| topic
+| Type of the exchange to publish log events to - only needed if the appender declares the exchange.
+See `declareExchange`.
+
+| routingKeyPattern
+| %c.%p
+| Log4j-style pattern format to use to generate a routing key.
```

Comment:
That's why I added "-style" - the other implementations use the same pattern.

Not sure how else it can be reworded without going into lots of details.